CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+### 1.4.0
+- Replace with request with cross-fetch for better client support
+- Update npm packages
+- Update server example output
+- Refactor HttpClient
+- Add test for minimial response
+- Add test for setting TLS options (key, cert)
+- Fix code coverage tool
+- Better KeepAlive support
+- NOTE: Breaking change to get at the raw HTTP objects. See `Client.httpFor()`
```

Comment:
This could use more detail, maybe something like "Breaking Change: replaced `Client.requestFor()` function for accessing raw HTTP objects with `Client.httpFor()` which returns an object containing both the request and the response."